utils/sql/db_update_manifest.txt

```diff
@@ -444,6 +444,7 @@
 9188|2022_07_14_zone_expansion_revert.sql|SHOW COLUMNS FROM `zone` LIKE 'expansion'|empty|
 9189|2022_07_10_character_task_rewarded.sql|SHOW COLUMNS FROM `character_tasks` LIKE 'was_rewarded'|empty|
 9190|2022_07_13_task_reward_points.sql|SHOW COLUMNS FROM `tasks` LIKE 'reward_points'|empty|
+9191|2022_07_28_gm_state_changes.sql|SHOW COLUMNS FROM `account` LIKE `invulnerable`|empty|
```

Comment:
Same typo I made in my last patch, need to surround the column name with single quotes not backticks